spec/models/job_spec.rb

```diff
@@ -30,9 +30,15 @@
     end
 
     it "allows run for any ref in refs params" do
-      job = FactoryGirl.create :job, project: project, refs: "master, staging"
+      job = FactoryGirl.create :job, project: project, refs: "master, staging, /testing.*/, /^unstable$/, /unstable-v[0-9]{1,}.[0-9]{1,}.[0-9]{1,}/"
```

Comment:
Line is too long. [148/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.